test-data/unit/check-modules.test

```diff
@@ -670,45 +670,45 @@ def f(x: str) -> None: pass
 [case testTypeCheckNamedModule]
 # cmd: mypy -m m.a
 [file m/__init__.py]
-None + 1
```

Comment:
It shouldn't change. 

But apart from flags added to tests, and tests *inside* typeshed, and `mypy mypy` tests, I can't find tests that have automatic `strict-optional`. Are you sure it was done to the pytest tests?